kuzzleio/documentation
# #450 Add algolia docsearch levels informations

## What does this PR do?
The new (and free) search engine Algolia docsearch, needs informations about the content of the documentation pages. And this one will crawl our site once a day in order to index all the pages. This informations needs to be organized in levels. for example :
* lvl 0: SDK
* lvl 1: Javascript 7.x
* lvl 2: Controllers
* lvl 3: document
* lvl 4: create

To allow algolia to find each level, we need to specify HTML selectors via a [configuration file](https://github.com/kuzzleio/docsearch-configs/blob/master/configs/kuzzle.json) that will have to be modified later via a PR on their own repository, hence the addition of the data-attributes `data-algolia-lvl="<lvl>"`.

Note that we have made the decision to add the SDK version to level 1, which means that for the other pages, level 1 will be `Core <kuzzleVersion>`.

### How should this be manually tested?
Browse through the documentation and look for the classes `algolia-lvl<x>` and check that the levels correspond to the desired path.
